### PR TITLE
Add ECMA 262 and TC39 Stage 3 proposals to the list of specs 

### PR DESCRIPTION
**Pull request by dontcallmedom**
do not merge for now - reffy is not ready to work with github markdown pages as "specs".

### CODE REVIEW

**Comment by dontcallmedom** [comment]
now ready

File: src/compute-shortname.js [hunk 1]
```diff
@@ -56,6 +56,18 @@ function computeShortname(url) {
         return whatwg[1];
     }
 
+    // Handle ECMAScript
+    if (url === "https://tc39.es/ecma262/") {
+      return "ecmascript";
+    }
```
Comment (tidoust) [inline src/compute-shortname.js:62]: Given that this only deals with one spec, could that appear in a `shortname` property in `specs.json` instead?

**Review by tidoust — COMMENTED** [review]
See inline.

PR mentions that Reffy is not ready to work with GitHub markdown pages as specs. Was it solved by restricting the list to "stage 3" proposals that don't use markdown?

**Comment by tidoust** [comment]
I added a couple of tests. I don't see any direct way to test retrieval of the title: I suppose TC39 proposals are not meant to stay as proposals forever so they don't seem to be good candidate URLs for tests, and the title for the main ECMAScript spec actually comes from Specref.

**Comment by dontcallmedom** [comment]
yes, with stage 3 only proposals, reffy seems to work reasonably well.

**Review by tidoust — APPROVED** [review]

**Pull request merged**